test/auto_ownership/configs/auto_ownership.csv

```diff
@@ -0,0 +1,64 @@
+Label,Description,Expression,0_CARS,1_CAR_1CV,1_CAR_1AV,2_CARS_2CV,2_CARS_2AV,2_CARS_1CV1AV,3_CARS_3CV,3_CARS_3AV,3_CARS_2CV1AV,3_CARS_1CV2AV,4_CARS_4CV
+,Boolean indicating that household is in SF county,"@np.where(df.county_id == ID_SAN_FRANCISCO, 1, 0)",1.233276051,,,,,,,,,,
+,alternative specific constant for 1 driver household,@df.num_drivers==1,-1.920143842,,,-0.934889684,-0.934889684,-0.934889684,-1.904733531,-1.904733531,-1.904733531,-1.904733531,-2.488003709
```

Comment:
Is it worth moving all of these coefficients to the auto_ownership_coeffs.csv file?  Also, it looks like the coefficients in that file aren't actually getting used for anything, so they should either get deleted or replaced.